code-challenges.js

```diff
@@ -9,61 +9,139 @@
 
 // Set one:
-const fruit1 = "apple"
-const fruit2 = "banana"
-// Expected outcome: "banana"
+// const fruit1 = "apple"
+// const fruit2 = "banana"
+// // Expected outcome: "banana"
 
-// Set two:
-const fruit3 = "cherry"
-const fruit4 = "kiwi"
-// Expected outcome: "cherry"
+// // Set two:
+// const fruit3 = "cherry"
+// const fruit4 = "kiwi"
+// // Expected outcome: "cherry"
 
 // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
 // Pseudo code:
+// input: num
+// output: return "num is below boiling point" when num is lesser than 212 (using conditional statement if),
+// return "num is at boiling point" when num is equal to 212 (using conditional statement else if),
+// return "num is above the boiling point" when above conditional statement is not met. (using conditional statement else)
+
+// Function name:boilingPointDeterminer
+const boilingPointDterminer = (num) => {
+if(num < 212){
+    return ` ${num} is below boiling point`
+} else if (num === 212){
+    return ` ${num} is at boiling point`
+} else {
+    return ` ${num} is above boiling point`
+}
+}
 
 const temperature1 = 42
 // Expected output: "42 is below boiling point"
+console.log(boilingPointDterminer(temperature1));
+// Output: 42 is below boiling point
 
 const temperature2 = 350
 // Expected output: "350 is above boiling point"
+console.log(boilingPointDterminer(temperature2));
+// Output: 350 is above boiling point
 
 const temperature3 = 212
 // Expected output: "212 is at boiling point"
+console.log(boilingPointDterminer(temperature3));
+// Output:212 is at boiling point
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
+// input:arrays1, arrays2
+// output:return length value of combine arrays.
+// (possible solution: create a function to combine arrays1 and arrays2 and using .length method to get the length value)
+// 
+const lengthCombine =(arrays1, arrays2) =>{
+return arrays1.concat(arrays2).length
+}
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
+console.log(lengthCombine(padres1984WorldSeriesRuns, padres1998WorldSeriesRuns));
+// Output: 9
+
+
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
+
 // Pseudo code:
+// Input: string1
+// Output: return reverse letters with string1.
+// (Possible solution is to create a function to use .split("") method to turn string1 into a array and use .reverse() method with to reverse letters then use .join("") method to turn it back to strings.)
+// function: reverseFunction
+
+
+const reverseFunction =(string1) => {
+    return (string1.split("").reverse().join(""))
+}
```

Comment:
Woohoo! Indentation looks great, now just make sure you get your spacing down! Also, no need to wrap the whole return in parentheses! Well done!